pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -187,14 +204,14 @@ const (
 	// successfully when the APIBinding is deleting
 	BindingResourceDeleteSuccess conditionsv1alpha1.ConditionType = "BindingResourceDeleteSuccess"
 
-	// PermissionClaimsAccepted is a condition for APIBinding that indicates that the permission claims were fully accepted or not.
-	PermissionClaimsAccepted conditionsv1alpha1.ConditionType = "PermissionClaimAccepted"
+	// PermissionClaimsValid is a condition for APIBinding that indicates that the permission claims were valid or not.
+	PermissionClaimsValid conditionsv1alpha1.ConditionType = "PermissionClaimsValid"
 
-	// IdentityMismatchClaimInvalidReason is used one or more claims have an identity mismatch, between what is bound and what is accepted.
-	IdentityMismatchClaimInvalidReason = "ClaimIdentityMismatch"
+	// InvalidPermissionClaimsReason indicates there were unexpected and/or invalid permission claims (e.g. due to
+	// identity mismatch).
+	InvalidPermissionClaimsReason = "InvalidPermissionClaims"
 
-	// UnknownPermissionClaimInvalidReason  is used when no idenitty mismatches exist, but we are unable to update the resources.
-	UnknownPermissionClaimInvalidReason = "Unknown"
+	PermissionClaimsApplied conditionsv1alpha1.ConditionType = "PermissionClaimsApplied"
```

Comment:
why do we need this? What does it mean? We should apply the claims even if some are not valid. So I don't think `PermissionClaimsValid` can be about applying.

Reply:
maybe I misread. If we have both types, that's fine.